app/src/components/market/common/market_top_details_open/index.tsx

```diff
@@ -35,6 +40,14 @@ interface Props {
 
 const MarketTopDetailsOpen: React.FC<Props> = (props: Props) => {
   const context = useConnectedWeb3Context()
+
+  const screenEvaluationEquation = window.innerWidth <= parseInt(theme.themeBreakPoints.sm)
+  const [isMobile, setIsMobile] = useState(screenEvaluationEquation)
+
+  useEffect(() => {
+    setIsMobile(screenEvaluationEquation)
+  }, [screenEvaluationEquation])
+
```

Comment:
@Mi-Lan ,
`isMobile` is not working well on my side.
If you inspect the window and decrease the width gradually, it doesn't work correctly.

![Screen Shot 2020-11-03 at 7 28 33 PM](https://user-images.githubusercontent.com/34900782/98055139-c8a9a780-1e0a-11eb-9b4c-88115ca70eb5.png)

Reply:
@liaojikunwork well if you wait couple of second it will work...i can put dom events for that but i think it would be overkill...and who would decrease window size on desktop to mobile...

Reply:
@Mi-Lan , that's right.
What is your thought, @pimato ?

Reply:
@Mi-Lan , please check this: https://usehooks.com/useWindowSize/

Reply:
@liaojikunwork i mentioned the same thing to kaden yesterday lets see maybe what he says @KadenZipfel